app/src/main/res/values/strings.xml

```diff
@@ -172,4 +172,6 @@
     <string name="cd_navdrawer_user_picture">User Profile Image</string>
     <string name="title_signing_needed">Sign-in Needed</string>
     <string name="signin">Sign-in</string>
+    <string name="feedback_message_required">Message required</string>
+    <string name="feedback_invalid_email">Input valid email address</string>

```

Comment:
@tasomaniac as far as I know, "Input valid email address" is an instruction to the user in a polite way, where as "Email is not valid" is like a slapping user and saying that hey input valid one :) Rest is up to any of the native speaker :)